cloud/google/node_init.sh

```diff
@@ -42,9 +42,9 @@ fi
 sudo bash ./${AGENT_INSTALL_SCRIPT}
 
 # Examine what kind of Docker image is on this machine to determine what to log.
-if docker images | grep ct-server; then
+if docker images | grep '/ct-server$'; then
   CT_LOGS_PREFIX="${DATA_DIR}/ctlog/logs/ct-server"
-elif docker images | grep ct-mirror; then
+elif docker images | grep '/ct-mirror$'; then
```

Comment:
Are you sure that 'docker images' output will have a line ending ct-mirror ?
When I tried, it didn't:

(on a mirror host)
$ sudo docker images
REPOSITORY TAG IMAGE ID CREATED SIZE
gcr.io/ct-mirror-pilot/ct-mirror test 7643c5ff45ac 11 days ago 273.4 MB
google/cadvisor latest f9ba08bafdea 12 weeks ago 57.34 MB

$ sudo docker images | grep '/ct-mirror$'
$

(on an etcd host)
$ sudo docker images
REPOSITORY                             TAG                 IMAGE ID            CREATED             SIZE
gcr.io/ct-mirror-pilot/etcd            test                bf26662588bc        3 days ago          194.9 MB
google/cadvisor                        latest              f9ba08bafdea        12 weeks ago        57.34 MB
gcr.io/google_containers/pause-amd64   3.0                 99e59f495ffa        13 months ago       746.9 kB
$ sudo docker images | grep '/ct-mirror$'
$ 

Same would apply to the ct-server regexp above.